models/Context.php

```diff
@@ -340,7 +342,7 @@ private function inheritPropertyRecursive($method, $class)
      */
     private function getParents($class)
     {
-        if ($class->parentClass === null || !isset($this->classes[$class->parentClass])) {
+        if ($class instanceof InterfaceDoc || $class->parentClass === null || !isset($this->classes[$class->parentClass])) {
```

Comment:
We have `interface ElementQueryInterface extends \yii\db\QueryInterface` and a method on that interface with an `@inheritDoc`:

```
/**
 * @inheritdoc
 *
 * @return ElementInterface|null The resulting element.
 */
public function one($db = null);
```

Eventually `Context->inheritMethodRecursive()` gets around to calling `$this->getParents($class)` where `$class` is of type `InterfaceDoc` (referring to our `ElementQueryInterface` and `$method` is referring to our `one()`.

Maybe the bug is that `inheritMethodRecursive()` shouldn't be calling `$this->getParents()` on interfaces?

Reply:
@angrybrad sorry that the pull request got lost. I'm getting over issues in this repository and trying to verify everything. I'm unsure how to reproduce the issue to fix it i.e. what code should I try to generate apidoc for in order to see the issue?